index.html

```diff
@@ -1,29 +1,250 @@
 
+          Admin </a></li>
+                            <li><a href="#"><i class="far fa-calendar"></i>              January 12,2019 </a></li>
+                            <li><i class="far fa-comment"></i> <a href="#">2 Comments </a></li>
+                        </ul>
+                    </div>
+                </div>
+                <br>
+                <a href="#">
+                    <h3>Woman claims husband wants to name baby girl after his ex-lover sparking. </h3>
+                </a>
+                <br>
+                <p>Tag: <a href="#">travel</a>, <a href="#">life style</a>, <a href="#">technology</a>, <a href="#">fashion</a></p>
+                <br>
+                <p>Over yielding doesn't so moved green saw meat hath fish he him from given yielding lesser cattle were fruitful lights. Given let have, lesser their made him above gathered dominion sixth. Creeping deep said can't called second. Air created
+                    seed heaven sixth created living</p>
+                <br>
+                <button class="butn bbtnw btnhvr">READ MORE   <i class="fa fa-arrow-right" aria-hidden="true"></i></button>
+            </div>
+            <div class="blog1">
+                <div class="imgc"> <img src="images/blog4.png" width="100%" height="100%" alt="blog one image">
+                    <div class="imginfo">
+                        <ul>
+                            <li><a href="#"><i class="far fa-user"></i>
 
+        Admin </a></li>
+                            <li><a href="#"><i class="far fa-calendar"></i>              January 12,2019 </a></li>
+                            <li><i class="far fa-comment"></i> <a href="#">2 Comments </a></li>
+                        </ul>
+                    </div>
+                </div>
+                <br>
+                <a href="#">
+                    <h3>Woman claims husband wants to name baby girl after his ex-lover sparking. </h3>
+                </a>
+                <br>
+                <p>Tag: <a href="#">travel</a>, <a href="#">life style</a>, <a href="#">technology</a>, <a href="#">fashion</a></p>
+                <br>
+                <p>Over yielding doesn't so moved green saw meat hath fish he him from given yielding lesser cattle were fruitful lights. Given let have, lesser their made him above gathered dominion sixth. Creeping deep said can't called second. Air created
+                    seed heaven sixth created living</p>
+                <br>
+                <button class="butn bbtnw btnhvr">READ MORE   <i class="fa fa-arrow-right" aria-hidden="true"></i></button>
+            </div>
 
+        </div>
+        <div id="asidepartcont">
+            <div class="newsletter newslsize">
+                <div class="boxcont">
+                    <h4>Newsletter</h4><br>
+                    <form>
+                        <input type="text" name="email" placeholder="Enter email" class="email">
```

Comment:
since this input is email you should give it `type="email"` instead of text

Reply:
okay i made it